package.json

```diff
@@ -25,10 +25,12 @@
     "start": "grunt dev",
     "test": "grunt test",
     "docs": "npm run docs:lint && npm run docs:api",
-    "docs:api": "jsdoc -r src/js -d docs/api -c .jsdoc.json",
+    "jsdoc": "jsdoc",
```

Comment:
seems outside of the scope of what scripts should do. I think its easy enough to do `./node_modules/.bin/jsdoc`